### ISSUE TITLE
[Bug] Old inline image missing alt text, no hint to attached file after import

### ISSUE THREAD

**Issue by bobo96run** [dotnetfactory/openproject-jira-importer#29]
As can be seen in https://github.com/dotnetfactory/openproject-jira-importer/pull/23#issuecomment-3589069472, inline images (whether in Jira issue description or comment body) are broken after import into OpenProject with this script, but at least we have the `alt` text with the file name, giving a useful hint to the attached file (which is properly imported) that should be there:

<img width="408" height="71" alt="Image" src="https://github.com/user-attachments/assets/3b2281fc-8775-4fab-b081-9498282334de" />

---

Unfortunately, while importing old issues, I noticed that all issues prior late March 2024 do not have that `alt` text...

Example issue around March 2024:

<img width="642" height="116" alt="Image" src="https://github.com/user-attachments/assets/bf56fef3-19fb-40cf-82ea-9a2cf8e5b97d" />

```html
<img src="/rest/api/3/attachment/content/23543" height="1165" width="1272" style="border: 0px solid black" class="op-uc-image">
```

Example issue around April 2024:

<img width="247" height="45" alt="Image" src="https://github.com/user-attachments/assets/1ceb66c4-30f5-4be3-a27b-8519af8461bf" />

```html
<img src="/rest/api/3/attachment/content/23575" alt="image-20240402-075705.png" height="380" width="1213" style="border: 0px solid black" class="op-uc-image">
```

So for this old content, we are back to square one, with no hint to what image is referred to in the description or comment.

**Comment by bobo96run** [comment]
Obviously, the integer part at the end of the image `src` attribute value is the Jira ID of the file (probably scoped to the Jira instance). But when this script imports the image/file into OP, we lose this value, and we have only the new file ID assigned by OP
https://github.com/dotnetfactory/openproject-jira-importer/blob/dd8a67a49420a1f4298ef5f5cc6c75de56633937/index.js#L252-L256
...so after this, the Jira ID is no longer associated to the OP file.

Maybe we could store that Jira ID in the file description, but I am not sure how OP displays it?
https://www.openproject.org/docs/api/endpoints/attachments/#create-work-package-attachment
> the body _must_ be a single JSON object, containing at least the `fileName` and optionally the attachments `description`.

Furthermore, the reader would need to inspect the broken image to get the `src` value...

In the Jira issue data, the file data has (among other) both the file name and the Jira ID, so it is definitely possible to make the association between the `src` value and the file (name), and add it back into the image (e.g. in the `alt` attribute, like for recent issues?).
```json
{
  "self": "https://<instance>.atlassian.net/rest/api/3/attachment/23543",
  "id": "23543",
  "filename": "Capture d’écran 2024-03-26 à 09.42.01.png"
}
```

But now, that means parsing and rewriting the description (or comment body) HTML content!

If we start doing that, we could as well replace the `src` value by the OP file path, hence fixing #14
